### ISSUE TITLE
Inconsistent build status messages for container builds

### ISSUE THREAD

**Issue by johnmcollier** [eclipse/codewind#1357]
Something I just noticed when deploying multiple Codewind-style projects on Che, we're using inconsistent messages to convey that a projects container is being built:
<img width="444" alt="Screen Shot 2019-12-03 at 5 26 06 PM" src="https://user-images.githubusercontent.com/6880023/70095163-12ce4b80-15f2-11ea-98b1-ab6a023f422e.png">

Most projects (generic docker and Node) use `Creating image`, Microprofile projects use `Building container image` and Swift uses `Creating image for application`. 

To be consistent, we should use a common message across all project types when building a container image, `Creating image` is short and concise and should do the job fine.

**Comment by ssh24** [comment]
FYI - Swift does have two different steps, docker build and docker application build, so it does require two different messages

**Comment by johnmcollier** [comment]
@ssh24 Good point. On Kube though there's only one container build, so we should make sure the status on Swift is consistent there too
